.github/workflows/main.yml

```diff
@@ -32,3 +32,13 @@ jobs:
         SPACE_REGION: ${{ secrets.Spaces_Region}}
         SPACE_ACCESS_KEY_ID: ${{ secrets.Spaces_Key }}
         SPACE_SECRET_ACCESS_KEY: ${{ secrets.Spaces_Secret }}
+    - name: Wait / Sleep
+      uses: kibertoad/wait-action@1.0.1
+      with:
+        time: '2m'
+    - name: Cloudflare Purge Cache
+      uses: jakejarvis/cloudflare-purge-action@v0.3.0
+      env:
+        CLOUDFLARE_ZONE: ${{ secrets.CLOUDFLARE_ZONE }}
+        CLOUDFLARE_TOKEN: ${{ secrets.CLOUDFLARE_TOKEN }}
+        PURGE_URLS: '["https://cdn.libretexts.net/github/ckeditor-binder-plugin/js/registerPlugin.min.js"]'
```

Comment:
Can we also add` registerPlugin.min.js.map` to the purge URLs?

Reply:
Sure, just append it within the array.

Reply:
Should I do it?